### PR TITLE
configured eslint and prettier.installed and set up husky

### PR DESCRIPTION
**Pull request by Adarshvarghese**
#78 
please check the screenshots
eslint
husky
eg
 i did not run commands shown in eg because  i think  it is given as an example .please correct me if i am wrong

### CODE REVIEW

**Review by anjali9265 — CHANGES_REQUESTED** [review]
Please add a config file for prettier specifying the following:
 1. Print width of 120
 2. Trailing comma
 3. Tab width of 2 spaces
 4. Semicolon
 5. Single quote as true
 
 Also in the Husky hooks add the check for lint and prettier.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^8.42.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.32.2",
```
Comment (anjali9265) [inline package.json:22]: Why do we have eslint-plugin-react package?

Comment (Adarshvarghese) [reply package.json]: it was added by mistake.

Comment (Adarshvarghese) [reply package.json]: i have removed it and pushed changes.Please let me know if there is anything else to be changed.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "extends": [
+        "standard",
+        "plugin:react/recommended","prettier"
+    ],
+    
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
```
Comment (anjali9265) [inline .eslintrc.json:15]: Please remove all react-related configurations. We don't need those as our project is purely HTML.

**Review by anjali9265 — CHANGES_REQUESTED** [review]

**Review by anjali9265 — APPROVED** [review]

**Pull request merged**